SignalsIntegrationTests/SignalsIntegrationTests/GetDataArgumentsTests.cs

```diff
@@ -0,0 +1,320 @@
+﻿using System;
+using Domain;
+using Dto.Conversions;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using SignalsIntegrationTests.Infrastructure;
+
+namespace SignalsIntegrationTests
+{
+    [TestClass]
+    public class GetDataArgumentsTests : TestsBase
+    {
+        private DateTime FromTimestamp { get; } = new DateTime(2029, 1, 1, 0, 0, 0, 0);
+        private DateTime ToTimestamp { get; } = new DateTime(2029, 1, 8, 0, 0, 0, 0);
```

Comment:
I think we should add separate issue for that, and test it per `MissingValuePolicy`. All those "download one sample older than from" etc. might result in some strange behavior in possible implementations.

Our is safe, thanks to TImeEnumerator :D